docs/install-include.rst

```diff
@@ -164,8 +164,13 @@ Select your preferences below and run the corresponding install commands:
 AutoGluon is modularized into `sub-modules <https://packaging.python.org/guides/packaging-namespace-packages/>`_ specialized for tabular, text, or image data. You can reduce the number of dependencies required by solely installing a specific sub-module via:  `python3 -m pip install <submodule>`, where `<submodule>` may be one of the following options:
 
 - `autogluon.tabular` - only functionality for tabular data (TabularPredictor)
+    - The default installation of `autogluon.tabular` standalone is a skeleton installation.
+    - Install via `pip install autogluon.tabular[all]` to get the same installation as via `pip install autogluon`
+    - Available optional dependencies: `lightgbm,catboost,xgboost,fastai`. These are included in `all`.
+    - Experimental optional dependency: `skex`. This will speedup KNN models by 25x in training and inference on CPU. Use `pip install autogluon.tabular[all,skex]` to enable, or `pip install "scikit-learn-intelex<2021.3"` after a standard installation of AutoGluon.
 - `autogluon.vision` - only functionality for computer vision (ImagePredictor, ObjectDetector)
 - `autogluon.text` - only functionality for natural language processing (TextPredictor)
 - `autogluon.core` - only core functionality (Searcher/Scheduler) useful for hyperparameter tuning of arbitrary code/models.
 - `autogluon.features` - only functionality for feature generation / feature preprocessing pipelines (primarily related to Tabular data).
 - `autogluon.extra` - miscellaneous extra functionality such as `Efficient Neural Architecture Search <tutorials/nas/enas_proxylessnas.html>`_
+- `autogluon.mxnet` - miscellaneous extra functionality for MXNet.
```

Comment:
"functionality for MXNet" -> "MXNet functionality"

Reply:
might be helpful to also show how to pip install a submodule from source

Reply:
added